### ISSUE TITLE
App gives error message upon starting 

### ISSUE THREAD

**Issue by CR1MS0N24** [saikou-app/saikou#236]
![Screenshot_2022-06-06-00-03-53-678_ani saikou beta](https://user-images.githubusercontent.com/87857932/172074146-1c22c64b-5878-4b1d-a88a-6a409552fef1.jpg)

**Comment by Shemar890** [comment]
![Screenshot_20220605-183236_Saikou .jpg](https://user-images.githubusercontent.com/106935450/172075143-4978b140-cf59-409a-a545-7b9e6ac6040d.jpg)

**Comment by CR1MS0N24** [comment]
That issue happens with your list mine doesn't even show anything

**Comment by Shemar890** [comment]
They need fix it tomorrow

**Comment by CR1MS0N24** [comment]
Oh

**Comment by CR1MS0N24** [comment]
But on another note are you experiencing any issues with using your anilist account to sign into other third party apps

**Comment by brahmkshatriya** [comment]
> But on another note are you experiencing any issues with using your anilist account to sign into other third party apps

Are you facing the first issue on other anilist apps?

**Comment by CR1MS0N24** [comment]
Yes basically any other app that scraped data from anilist wasn't working for the time being but somehow it fixed itself

**Comment by brahmkshatriya** [comment]
So it was an anilist side problem, where I can do nothing
I'll just change error ig

**Comment by Shemar890** [comment]
Need to be fixed

**Comment by Shemar890** [comment]
No

**Comment by Manxchilz098** [comment]
Hope it fix soon then I thinks it's all great again
